bitnami/kafka/Chart.yaml

```
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 16.3.0
+version: 16.3.1
```

Since this is adding a new feature, could we increase the minor instead? E.g.:

```suggestion
version: 16.4.0
```

---

Done!